examples/step-64/step-64.cc

```diff
@@ -131,7 +131,9 @@ namespace Step64
       const typename CUDAWrappers::MatrixFree<dim, double>::Data *gpu_data,
       double *                                                    coef,
       int                                                         cell)
-      : coef(coef)
+      : gpu_data(gpu_data)
+      , coef(coef)
+      , cell(cell)
```

Comment:
We forgot to initialize the members resulting in segmentation faults for the current `master`.

Reply:
Oops :-)